app/services/policies/policies_list_service.rb

```diff
@@ -1,11 +1,57 @@
 # frozen_string_literal: true
 
 require 'jsonclient'
+require 'set'
 
 class Policies::PoliciesListService
-
-  def self.call
+  def self.call(account)
     response = ::JSONClient.get(ThreeScale.config.sandbox_proxy.apicast_registry_url)
-    response.body.symbolize_keys[:policies] if response.ok?
+    list = response.ok? ? PolicyList.from_hash(response.body['policies']) : PolicyList.new
```

Comment:
hmmm, I think if it somehow fails fetching the policies, we should do something about it, not just respond with the custom ones. WDYT?

Reply:
What could be the reason of response not being ok? If it was error (not an unauthorized request for instance), I'd say we should propagate it somehow

Reply:
many reasons, those are fetch from apicast ... and apicast could be anywhere

Reply:
Yes you are right @didierofrivia 
Before it was just returning `nil` but no tests why. If `null` corresponds to something expected by the editor then I will gladly change it back and add a test

Reply:
So @didierofrivia @josemigallas lets decide what to do or do it in another PR if we are not sure.